paimon-web-server/pom.xml

```diff
@@ -168,6 +169,12 @@ under the License.
             <version>${hadoop.version}</version>
             <scope>provided</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.github.xiaoymin</groupId>
```

Comment:
Why use this dependency?

Reply:
> Why use this dependency?

This provides the ability of Api Documentation for front-end development.

Reply:
Does this feature need this dependency?